feat(storage): add doc on hosting health data MTA-6713 (#5786)
scaleway/docs-content: pages/block-storage/reference-content/storage-shared-responsibility-model.mdx

---
title: Scaleway storage services shared responsibility model
description: Learn about the shared responsibility model for Scaleway Block Storage and Scaleway Object Storage, outlining the roles of Scaleway and users in managing data encryption security, and healthcare data hosting (HDS) compliance.
tags: storage shared responsibility model compliance hds health data hosting
dates:
  created: 2025-11-12
  validation: 2025-11-12
---

This document outlines the roles and responsibilities for maintaining and securing your Scaleway storage services, Block Storage and Object Storage including Glacier. Our shared responsibility model clarifies the division of duties between Scaleway and our users, ensuring clarity in managing availability, backups, configurations, and security measures for your storage solutions. By understanding this shared responsibility, you can optimize the performance, reliability, and security of your Scaleway storage services.

## Service Provision

Scaleway is responsible for the operational maintenance of all components essential to providing the Service. These include:

> *   **Physical Infrastructure:** Managing the physical sites that house the hardware infrastructure used for the Service.

Reviewer: ```suggestion
*   **Physical Infrastructure:** Managing the physical sites that house the hardware infrastructure used for the service.
```

*   **Hardware Infrastructure:** Maintaining the underlying hardware. Monitors resource utilization rate and updates its capacity plan.

*   **Virtual Infrastructure:** Ensuring the functionality of the virtualized environment.

*   **Hosting Platform:** Operating the application and database hosting platform.

*   **Network:** Establishes storage space connectivity in its default configuration.

*   **Applications and Databases:** Maintaining the applications and databases themselves.

Furthermore, Scaleway procures the necessary licenses and usage rights for any third-party solutions that are either used by Scaleway in delivering the Service or made available to the client as part of the Service. It provides the client with necessary information regarding the characteristics and conditions of use of the Service. It also plans and implements updates to the various Service components also considering third-party products used within the Service.

You are responsible of ensuring that users comply with the Service's terms of use.

Scaleway is responsible for monitoring, managing, and forecasting the Services APIs and physical clusters capacity.

> You are responsible for informing Scaleway ahead of time for significant changes in your storage capacity forecast.

Reviewer 2: ```suggestion
You are responsible for informing Scaleway ahead of time of significant changes in your storage capacity forecast.
```

## Product Resiliency

### Availability and Steadiness

Scaleway ensures high availability through resilient infrastructure, technical and organizational measures, including monitoring service health and incident detection for the data hosted within the Service. Scaleway provides and maintains the operational condition of the control plane and APIs.

Scaleway is responsible for:

*   Implementing resiliency for the Services at least within an Availability Zone (AZ), and for multi-AZ storage tiers only across multiple Availability Zones.

*   Monitoring service health, performance, and balancing the load of use of the Service.

You are responsible for:

*   Implementing technical and organizational measures to ensure the continuity of your activities, taking into account the conditions of use and characteristics of the Service, including the nature and criticality of data stored.

*   Configuring the Service in accordance with your needs, particularly in terms of availability.

*   Monitoring and notifying breaches of Service Level Agreement concerning the availability of the service.

#### Performance

> You are responsible for optimizing your applications' use of Storage Services. Scaleway monitors and optimize infrastructure-level performance, but note that:

Reviewer 2: ```suggestion
You are responsible for optimizing your applications' use of Storage Services. Scaleway monitors and optimizes infrastructure-level performance, but note that:
```

*   Glacier cold storage data access requires object restoration and can incur delays,

*   For Object Storage Service, multipart uploads and concurrent requests should be used for large files.

**Data Integrity**

For all Storage Services, you are responsible for:

*   Verifying the successful completion of read and write operations.

*   Verifying the successful completion and integrity of backups.

For the Object Storage Service, Scaleway provides a checksum of each uploaded object allowing the client to perform integrity control or control the integrity of the checksum provided to the client at the time of upload, if applicable.

You are responsible for:

*   Verifying the successful completion of the upload via a checksum control.

*   Controlling the availability and integrity of objects, and restoring damaged objects.   

## Backups and Replication 

While Scaleway provides internal resiliency mechanisms and features, it is your responsibility to implement a backup strategy appropriate to your needs and the criticality of your activities, including:

*   External backups.

*   Replication to other regions (as allowed by regulations).

> *   Management over your backups and snapshots, and regular control of their integrity.  

Reviewer 2: ```suggestion
*   Management of your backups and snapshots, and regular verification of their integrity.  
```

  
Internal resilience does not protect against accidental deletions or application-level corruption.

## Configuration and Management

Scaleway is responsible for the configuration of API and dataplane settings.

You are responsible for managing the state of your storage spaces and objects. This includes monitoring and configuring service-level features and policies available for your storage space.

You are responsible for:

*   Creating storage spaces (buckets, volumes)

*   Provisioning additional space or storage spaces according to your needs

> *   Managing volumes attachment, detachment and deletion according to your needs

Reviewer 2: ```suggestion
*   Managing volume attachment, detachment, and deletion according to your needs
```

*   Managing snapshots and snapshots deletion according to your needs

### Object Storage Class & Lifecycle Rules

> You are responsible for selecting appropriate storage classes, cleanup unnecessary parts or data, and using lifecycle rules according to your needs. Note that a delay may occur if transitioning or expiring a high number of objects through lifecycle rules per day.

Reviewer 2: ```suggestion
You are responsible for selecting appropriate storage classes, cleaning up unnecessary parts or data, and using lifecycle rules according to your needs. Note that a delay may occur if transitioning or expiring a high number of objects through lifecycle rules per day.
```

### Access Control 

Scaleway provides versioning, IAM, and specific Service-level access control tools (ACLs, bucket policies.)

> You are responsible for managing authorizations and access of your personnel to the Service (Console, API and storage spaces), also ensuring the security of your personnel's authentication means. Your responsibilities include:

Reviewer 2: ```suggestion
You are responsible for managing authorizations and access of your personnel to the Service (Console, API and storage spaces), and for ensuring the security of your personnel's authentication means. Your responsibilities include:
```

*   Ensuring public visibility settings align with your intentions.

*   Regularly reviewing access rules and permissions.

*   Activating two-factor authentication (2FA).

*   Configuring bucket policies (access limitation under certain conditions such as IP with white list and black list (allow or deny, IP range)).

> Scaleway enforces your configurations but does not intervene in their definition or maintenance.

Reviewer 2: ```suggestion
Scaleway enforces your configurations but does not intervene in its definition or maintenance.
```
